Core/HLE/ReplaceTables.cpp

```diff
@@ -1183,6 +1210,29 @@ static int Hook_marvelalliance1_copy_after() {
 	return 0;
 }
 
+static int Hook_starocean_clear_framebuf_before() {
+	//skipGPUReplacements |= (int)GPUReplacementSkip::MEMSET;
```

Comment:
Can you add a comment on why this is left behind?

Reply:
Derp, I was validating that the counter was working and forgot to uncomment this.

-[Unknown]